hack/kustomize/deploy.sh

```diff
@@ -0,0 +1,100 @@
+#! /bin/bash
+
+# This script automates the deployment of the collector to a specific k8s cluster
+
+DEFAULT_IMAGE_NAME="wavefronthq\/wavefront-kubernetes-collector"
+DEFAULT_VERSION="1.2.6"
+DEFAULT_USE_CLASSIC_PROMETHEUS=false
+DEFAULT_FLUSH_ONCE=false
+DEFAULT_FLUSH_INTERVAL=30
+DEFAULT_COLLECTION_INTERVAL=60
+
+if [[ -z ${FLUSH_ONCE} ]] ; then
```

Comment:
So these are just environment vars that one can set?

Reply:
For now, we wanted to do the simplest thing that worked until it got too painful. Possible next steps once it gets to painful:
1. Use a overrides yaml file and  write a bash or go program to merge an overrides yaml file into collector yaml file
2. Look into other ways would could get the kustomize ConfigMapGenerator to work correctly or see if we can get kustomize patches to update the override values